extensions/wikia/Recirculation/js/plista.js

```diff
@@ -0,0 +1,70 @@
+define('ext.wikia.recirculation.plista', ['jquery'], function ($) {
+	function shouldFetchPlista(items) {
+		var index = items.findIndex(function (item) {
+			return item.presented_by;
+		});
+
+		return true;//index === -1 && ['AU', 'NZ'].indexOf(Geo.getCountryCode()) > -1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)